_data/projects/ReactStyleguidist.yml

```diff
@@ -9,4 +10,5 @@ tags:
 - UI
 upforgrabs:
   name: good first issue
-  link: https://github.com/styleguidist/react-styleguidist/issues?q=is%3Aissue+is%3Aopen+label%3A%22good%20first%20issue%22
+  link: >-
+    https://github.com/styleguidist/react-styleguidist/issues?q=is%3Aissue+is%3Aopen+label%3A%22good%20first%20issue%22
```

Comment:
This change can be distilled down to a `label` URL, but I have work queued up to tackle these so I won't address it here